HACKING

```diff
@@ -40,36 +40,45 @@ CHECK_MICRO_VERSION=1
 
 (Remember to update both the AC_INIT line and each of the CHECK_*_VERSION fields).
 
-3) Update the header in the NEWS file to mention the release:
+4) Update version in .travs.yml:
+
+env:
+  global:
+    - MY_PV=0.12.0
+
+
+4) Update the header in the NEWS file to mention the release:
 
 Sun Aug 2, 2015: Released Check 0.10.1
   2015-08-02 19:21:14 +0000
   based on hash f399542eeceb97703bca496b68bb39044e8baa01
 
-4) Update index.html mentioning the release. Look for the following:
+5) Update index.html mentioning the release. Look for the following:
 <!-- Update this section during a release -->
 
-5) Attempt to build the release locally
+6) Attempt to build the release locally
 
 $ autoreconf -i
 $ ./configure
 $ make distcheck
 
 If this passes, a tarball with the current release number will be in the current
-directory. Make note of this, as it will be uploaded to GitHub later.
+directory.
 
-6) Commit the changes to configure.ac, NEWS, and index.html to the Check project's
-master branch, with the commit message:
+7) Commit the changes to configure.ac, NEWS, and index.html to the Check 
+project's master branch, with the commit message:
 "Update for release"
 
-7) Log On to GitHub and navigate to:
+8) Push the commit to Github
 
-https://github.com/libcheck/check/releases
+9) If the build passes, tag the commit with the current release number 
+and push the tag to Github.
```

Comment:
Please put an example of creating the tag using git. The tag mentioned in the example from step 2 is 0.10.1, so use that in the example.